iavl/cli.py

```diff
@@ -285,5 +306,39 @@ def fast_rollback(
         print(f"update latest version to {target}")
 
 
+@cli.command()
+@click.option(
+    "--db", help="path to application.db", type=click.Path(exists=True), required=True
+)
+@click.option(
+    "--version",
+    help="the version to query, default to latest version if not provided",
+    type=click.INT,
+)
+@click.option(
+    "--include-prev-version",
+    help="include the previous version to compare",
+    is_flag=True,
+    default=False,
+)
+@click.option("--store", "-s")
+def visualize(db, version, store=None, include_prev_version=False):
```

Comment:
optional store seems not correct

Reply:
without passing store, it assume it's not a prefixed db, which is the case for the unit test db.